content/applications/sales/point_of_sale/shop/salesorder.rst

```diff
@@ -0,0 +1,45 @@
+==================
+Sales order in PoS
+==================
+
+Point of sale management can expose businesses to certain difficulties, such as the lack of a
+particular product in-store. To prevent losing a potential client, they need a way to order products
+directly from their shop.
+
+Fortunately, the point of sale application is fully integrated with the sales application, meaning
+that one can create a sales order and find it in the pos. Then, they can pay for it, either
+partially or completely, from the pos.
+
+Select a sales order
+====================
+
+From the **Point of Sale** application, open a new session. Then, click on
```

Comment:
I believe this one needs the `guilabel` role too since it's part of the interface and of the menus selection.